lib/hammer_cli_foreman/realm.rb

```diff
@@ -0,0 +1,53 @@
+module HammerCLIForeman
+  class Realm < HammerCLIForeman::Command
+
+    resource :realms
+
+    class ListCommand < HammerCLIForeman::ListCommand
+      output do
+        field :id, _("Id")
+        field :name, _("Name")
+      end
+
+      build_options
+    end
+
+
+    class InfoCommand < HammerCLIForeman::InfoCommand
+      output ListCommand.output_definition do
+        field :realm_proxy_id, _("Realm proxy id")
```

Comment:
I checked the api and proxy name isn't contained in the show reply at the moment. Disregard my previous post.